Makefile

```diff
@@ -19,26 +19,25 @@ verify: vaultwarden.s9pk $(S9PK_PATH)
 install: vaultwarden.s9pk
 	embassy-cli package install vaultwarden.s9pk
 
-vaultwarden.s9pk: manifest.yaml LICENSE image.tar instructions.md icon.png $(ASSET_PATHS)
+vaultwarden.s9pk: manifest.yaml LICENSE image.tar instructions.md icon.png $(ASSET_PATHS) scripts/embassy.js
 	embassy-sdk pack
 
 instructions.md: docs/instructions.md $(DOC_ASSETS)
 	cd docs && md-packer < instructions.md > ../instructions.md
 
 image.tar: Dockerfile $(VAULTWARDEN_SRC) docker_entrypoint.sh check-web.sh manifest.yaml
 	cp ./docker_entrypoint.sh ./vaultwarden/docker_entrypoint.sh
-	cp ./check-web.sh ./vaultwarden/check-web.sh
```

Comment:
why did we need to remove this?

Reply:
Because now we are using the js version of the health checks

Reply:
Also, going to remove the file too since we don't need